src/world.cpp

```diff
@@ -124,6 +125,7 @@ bool World::init(vec2 screen)
 	m_display_screen.init();
 	m_level_manager.init();
 	m_high_scores.init();
+	m_weather.init();
 
 	return m_traffic_cop.init();
 }

```

Comment:
can you add the `weather.destroy()` to `World::destroy()`? (do we ever actually use `World::destroy()`?)

Reply:
it's right below this line, but github won't let me comment on it for some reason

Reply:
We don't ever use world::destroy, but I will add it anyway!